lib/web_ui/lib/src/engine/text/line_breaker.dart

```diff
@@ -21,10 +21,39 @@ enum LineBreakType {
 }
 
 /// Acts as a tuple that encapsulates information about a line break.
+///
+/// It contains multiple indices that are helpful when it comes to measuring the
+/// width of a line of text.
+///
+/// [indexWithoutTrailingSpaces] <= [indexWithoutTrailingNewlines] <= [index]
 class LineBreakResult {
-  LineBreakResult(this.index, this.type);
-
+  LineBreakResult(
+    this.index,
+    this.indexWithoutTrailingNewlines,
+    this.indexWithoutTrailingSpaces,
+    this.type,
+  );
```

Comment:
Can this be `const`?

Reply:
I'll make it const but I don't see the benefit since we only construct it at runtime.